datafusion/sqllogictest/test_files/update.slt

```diff
@@ -26,30 +26,54 @@ create table t1(a int, b varchar, c double, d int);
 statement ok
 set datafusion.optimizer.max_passes = 0;
 
-query T
+query TT
 explain update t1 set a=1, b=2, c=3.0, d=NULL;
 ----
-This feature is not implemented: Unsupported logical plan: Dml(Update)
+logical_plan
+01)Dml: op=[Update] table=[t1]
+02)--Projection: CAST(Int64(1) AS Int32) AS a, CAST(Int64(2) AS Utf8) AS b, Float64(3) AS c, CAST(NULL AS Int32) AS d
+03)----TableScan: t1
+physical_plan_error This feature is not implemented: Unsupported logical plan: Dml(Update)
 
-query T
+query TT
 explain update t1 set a=c+1, b=a, c=c+1.0, d=b;
 ----
-This feature is not implemented: Unsupported logical plan: Dml(Update)
+logical_plan
+01)Dml: op=[Update] table=[t1]
+02)--Projection: CAST(t1.c + CAST(Int64(1) AS Float64) AS Int32) AS a, CAST(t1.a AS Utf8) AS b, t1.c + Float64(1) AS c, CAST(t1.b AS Int32) AS d
+03)----TableScan: t1
+physical_plan_error This feature is not implemented: Unsupported logical plan: Dml(Update)
 
 statement ok
 create table t2(a int, b varchar, c double, d int);
 
 ## set from subquery
-query T
+query TT
 explain update t1 set b = (select max(b) from t2 where t1.a = t2.a)
 ----
-This feature is not implemented: Physical plan does not support logical expression ScalarSubquery(<subquery>)
+logical_plan
+01)Dml: op=[Update] table=[t1]
+02)--Projection: t1.a AS a, (<subquery>) AS b, t1.c AS c, t1.d AS d
+03)----Subquery:
+04)------Projection: max(t2.b)
+05)--------Aggregate: groupBy=[[]], aggr=[[max(t2.b)]]
+06)----------Filter: outer_ref(t1.a) = t2.a
+07)------------TableScan: t2
+08)----TableScan: t1
+physical_plan_error This feature is not implemented: Physical plan does not support logical expression ScalarSubquery(<subquery>)
 
 # set from other table
-query T
+query TT
 explain update t1 set b = t2.b, c = t2.a, d = 1 from t2 where t1.a = t2.a and t1.b > 'foo' and t2.c > 1.0;
 ----
-This feature is not implemented: Unsupported logical plan: Dml(Update)
+logical_plan
+01)Dml: op=[Update] table=[t1]
+02)--Projection: t1.a AS a, t2.b AS b, CAST(t2.a AS Float64) AS c, CAST(Int64(1) AS Int32) AS d
+03)----Filter: t1.a = t2.a AND t1.b > Utf8("foo") AND t2.c > Float64(1)
+04)------Cross Join: 
+05)--------TableScan: t1
+06)--------TableScan: t2
+physical_plan_error This feature is not implemented: Unsupported logical plan: Dml(Update)
```

Comment:
this way we still see the logical explain, but now it is also clear there is a physical plan error